dev/tasks/r/github.macos.autobrew.yml

```diff
@@ -64,6 +64,10 @@ jobs:
             any::rcmdcheck
             any::sys
             any::readr
+      - name: Clean up self-hosted tmp 
+        if: matrix.platform == 'macos-10.13'
+        shell: bash
+        run: rm -rf /Users/voltrondata/tmp/apache-arrow--git
```

Comment:
Should we use something like `$HOME/tmp` for future-proofing (not sure how it should be quoted in YAML, though)?